frontend/src/models/behaviors.js

```diff
@@ -26,3 +27,111 @@ export class RandomCardBehavior {
     return null;
   }
 }
+
+export class BasicRuleBasedBehaviour {
+  announcementToMake(possibleAnnouncements) {
+    // ToDo announce by "goodies"
+    /**
+     * for instance
+     * only one fehlsuit
+     * high number of trumps
+     * high trumps, such as 2x ten of hearts
+     * blank ace and starting
+     */
+    return null;
+  }
+
+  cardToPlay(hand, trick, memory) {
+    let baseCard = trick.baseCard();
+    if (!baseCard) {
+      /** It's our turn. Decide how to deal with cards */
+      return this.startingRule(hand, memory);
+    }
+    if (!baseCard.isTrump()) {
+      /** We need to serve a fehl card */
+      return this.fehlRule(hand, trick, memory);
+    }
+    // ToDo how to play if not starting or mustn't serve fehl
+    // i'm thinking of something working with expectation value
+    return sample(playableCards(hand.cards, baseCard));
+  }
+
+  startingRule(hand, memory) {
+    for (const suit of [suits.clubs, suits.spades, suits.hearts]) {
+      if (hand.hasBlankAce(suit) && !memory.fehlSuitPlayedBefore(suit)) {
+        return hand.fehlCardsBySuit(suit)[0];
+      }
+    }
+    // ToDo check if we know with whom we play and if we want to play a strategy
+    return sample(playableCards(hand.cards, undefined));
+  }
+
+  fehlRule(hand, trick, memory) {
+    let baseCard = trick.baseCard();
+    if (hand.mustServeSuit(baseCard.suit)) {
+      let fehlCards = hand.fehlCardsBySuit(baseCard.suit);
+      let highest = fehlCards[0];
+      let lowest = fehlCards.reverse()[0];
+
+      if (memory.fehlSuitPlayedBefore(baseCard.suit)) {
+        return lowest;
+      } else {
+        if (
+          trick.highestCard().card.value < highest.value &&
+          highest.value === 11 &&
+          fehlCards.length < 4
+        ) {
+          return highest;
+        } else {
+          return lowest;
+        }
+      }
+    } else {
+      let hUtils = new HandUtils(hand);
+      let usefulTrump = hUtils.getUsefulTrumpForFehlTrick(trick);
+      return usefulTrump
+        ? usefulTrump
+        : sample(playableCards(hand.cards, undefined));
+    }
+  }
+}
+
+/**
+ * This class contains biased opinions, therefore it's not part of hand
+ */
+export class HandUtils {
+  constructor(hand) {
+    this.hand = hand;
+  }
+
+  getUsefulTrumpForFehlTrick(trick) {
```

Comment:
We have this "rule": "Niemals unterm Fuchs stechen / Don't trump lower than a fox". I only trump with king of diamonds if chances are very unlikely that somebody might get higher (for example if I'm the last player in the trick). This is a very opinionated move to be honest, but it's quite convinced when you loose youre ten of diamonds to a fox.

Reply:
Ha, I suspected this would be some kind of "nie unterm Fuchs stechen" but then was confused why playing a ten of diamonds (which is of course lower than a Fuchs) would be returned :)